aiodocker/docker.py

```diff
@@ -149,6 +158,23 @@ def _endpoint(self, path):
         data = await Docker._result(response, 'json')
         return data
 
+    @staticmethod
+    def _docker_machine_ssl_context():
+        """Create a SSLContext object using DOCKER_* env vars.
+
+        """
+        context = ssl.SSLContext(PROTOCOL_TLS)
+        context.set_ciphers(ssl._RESTRICTED_SERVER_CIPHERS)
+        certs_path = os.environ.get('DOCKER_CERT_PATH', None)
+        if certs_path is None:
+            raise ValueError('Cannot create ssl context, DOCKER_CERT_PATH is not set!')
+
+        def file_path(file):
+            return os.path.join(certs_path, file)
```

Comment:
Could be improved using pathlib instead of adding an inner function since aiodocker now targets Python 3.6+.
After `certs_path = Path(certs_path)`, you could just `cafile=certs_path / 'ca.pem'` in the argument and so on.